README.md

```diff
@@ -1 +1,206 @@
+
+
+- **[use linters]**
+- **[Displays a list of books]**
+- **[Allows users to add new books to the list]**
+- **[Allows users to remove books from the list]**
+- **[Persists book data using local storage]**
+
+
+
+
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+ <p><a href="#">DEMO COMMING SOON </a></p>
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+- you need a laptop.
+- you need github installed on it.
+- You need a basic knowledge of Html and Css and JavaScript
+
+
+### Setup
+
+Clone this repository to your desired folder:
+- cd my-folder
+- git clone https://github.com/zbahati/wesome-books-using-module
+
+
+### Install
+
+Install this project with:
+
+
+- cd my-project
+- npm install .
+
+
+### Usage
+
+To use the Awesome Books website, simply open the index.html file in your web browser or use git pages of this project, The website will load and display a list of books. To add a new book, click the "Add Book" button by fill out the form with the book's title and author. To remove a book, click the "Remove" button next to the book you wish to delete
+
+
+### Run tests
+
+To run tests, run the following command:
+
+- npx stylelint "**/*.{css,scss}"
+- npx hint .
+- npx eslint .
+
+### Deployment
+
+You can deploy this project using: github page's
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+
+
+👤 **Bahati zirimwabagabo**
+
+- GitHub: [zbahati](https://github.com/zbahati)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/zirimwabagabo-bahati)
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+
+- [ ] **[Nav menu toggle on mobile devices]**
+- [ ] **[Add server]**
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- SUPPORT -->
+
+## ⭐️ Show your support <a name="support"></a>
+
+
+show me your support by following me on github and giving me a star please .
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+I would like to thank Microverse for guidance and support in this project and always.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LICENSE -->
+
+## 📝 License <a name="license"></a>
+
+COMING SOON
+
```

Comment:
- [ ] Kindly add the license link as it is required and it is the link that grants legal permit to use this repo.

Reply:
I also add license.md to the file to make sure that my project has license in it